packages/sanity/src/core/releases/tool/components/ReleaseDocumentPreview.tsx

```diff
@@ -40,9 +40,7 @@ export function ReleaseDocumentPreview({
               id: getPublishedId(documentId),
               type: documentTypeName,
             }}
-            searchParams={[
-              ['perspective', `bundle.${getBundleIdFromReleaseDocumentId(releaseId)}`],
-            ]}
+            searchParams={[['perspective', getBundleIdFromReleaseDocumentId(releaseId)]]}
```

Comment:
NIT: Maybe rename also `getBundleIdFromReleaseDocumentId` to `getReleaseNameFromReleaseId` ?

Reply:
Now that I think of it, we should strive for consistency in naming these `releaseId` vs `releaseDocumentId`, so I'll rename `selectedReleaseName` to `selectedReleaseId` and this one to `getReleaseIdFromReleaseDocumentId`

Reply:
I've been thinking of name and id as different things in releases.
The id for me has been the string that starts with `_.releases.<name>` and the name has been the value we have in the `document.name`. Are we aligned on this? 

So: 
- id: `_.releases.r123`
- name: `r123`

Reply:
Another use case for branded types ;)